hoomd/operations.py

```diff
@@ -117,29 +213,76 @@ def integrator(self, op):
 
     @property
     def updaters(self):
+        """list[`hoomd.operation.Updater`]: A list of updater operations.
+
+        Holds the list of updaters associated with this collection. The list can
+        be modified as a standard Python list.
+        """
         return self._updaters
 
     @property
     def analyzers(self):
+        """list[`hoomd.operation.Analzyer`]: A list of analyzer operations.
+
+        Holds the list of analyzers associated with this collection. The list
+        can be modified as a standard Python list.
+        """
         return self._analyzers
 
     @property
     def tuners(self):
+        """list[`hoomd.operation.Tuner`]: A list of tuner operations.
+
+        Holds the list of tuners associated with this collection. The list can be
+        modified as a standard Python list.
+        """
         return self._tuners
 
     def __iadd__(self, operation):
+        """Works the same as `Operations.add`.
+
+        Args:
+            operation (``operation``): A HOOMD-blue updater, analyzers, compute,
+                tuner, or integrator to add to the object.
+        """
         self.add(operation)
+        return self
 
     def remove(self, operation):
```

Comment:
I'm confused by the long distance between "add" and "remove." Shouldn't these methods be near each other?